rasa/core/training/story_conflict.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, sliced_states: List[State]) -> None:
 
         self._sliced_states = sliced_states
         # A list of actions that all follow from the same state.
-        self._conflicting_actions: defaultdict[Text, List[Text]] = defaultdict(
+        self._conflicting_actions: Dict[Text, List[Text]] = defaultdict(
```

Comment:
([deprecated in 3.9](https://docs.python.org/3/library/typing.html#typing.DefaultDict))

Reply:
ah you're right... but I needed to do this because a test failed to run and complained that defaultdict didn't support `[...]`. Let me look into it again